core/src/main/java/bisq/core/btc/setup/WalletsSetup.java

```diff
@@ -409,6 +420,64 @@ private void configPeerNodes(@Nullable Socks5Proxy proxy) {
         networkConfig.proposePeers(peers);
     }
 
+    ///////////////////////////////////////////////////////////////////////////////////////////
+    // Peer configuration check
+    ///////////////////////////////////////////////////////////////////////////////////////////
+
+    /* Provided a Peer and its VersionMessage, will check and handle Peer's advertised
+     * configuration. The VersionMessage is provided separately, because it is not
+     * always available and this is meant to be called from a listener that is triggered
+     * when the VersionMessage becomes available.
+     */
+    private void handleConfiguration(Peer peer, VersionMessage versionMessage) {
+        var wellConfigured = isWellConfigured(versionMessage);
+        if (wellConfigured) {
```

Comment:
A temp variable is not necessary imo, the method call is already very concise and readable
```suggestion
        if (isWellConfigured(versionMessage)) {
```

Reply:
A variable defining the condition is a style I use (and propagate) and, even if the called method is descriptive as well, I still use it. One reason is consistency of style. Another, the if statement doesn't care that the node being well configured is predicated on the version message (that's an implemenation detail), so there's no reason to put that in the if statement. I would be against your suggestion, because it optimizes for terseness at the cost of readability.